### PR TITLE
Print signed message for with signing with two signatures - Closes #385

### LINKED ISSUE #385: Print signed message for with signing with two signatures
Currently the print function only supports one publicKey and one signature. 

The signMessageWithTwoPublicKeys should also have a print function or the old ones support this format.

### PR DESCRIPTION
**Pull request by alepop**
closes #385

### CODE REVIEW

**Comment by willclarktech** [comment]
Hey @alepop, thanks for the PR. Can you think of a use case for calling `printSignedMessage` rather than `printSignedMessageWithTwoSecrets` on a message with a second signature? I was imagining `printSignedMessage` would just check if `secondSignature` is present and conditionally include the relevant sections. Then we only need one function, and whatever's calling it doesn't need to know that messages with one vs two signatures get printed differently.

**Comment by alepop** [comment]
@willclarktech when I started implementing this feature I thought the same. In my opinion than function is easier then better (avoid if statements as much as possible). But if you think that the better way is to implement it in one function then I do it.

**Comment by willclarktech** [comment]
@alepop I think so. That way we can also put `signedMessage` and `result` in `const`s rather than having to use `let` in `signAndPrintMessage`. We don't need any `if` statements in `printSignedMessage`, we can just conditionally include elements in the `outputArray`:
```js
const outputArray = [
	signedMessageHeader,
	messageHeader,
	message,
	publicKeyHeader,
	publicKey,
	secondPublicKey ? secondPublicKeyHeader : null,
	secondPublicKey,
	signatureHeader,
	signature,
	secondSignature ? secondSignatureHeader : null,
	secondSignature,
	signatureFooter,
].filter(Boolean);
```

**Comment by alepop** [comment]
@willclarktech agree. I'll fix it.

File: src/crypto/sign.js [hunk 1]
```diff
@@ -148,27 +156,28 @@ export function verifyMessageWithTwoPublicKeys(
  * @param message
  * @param signedMessage
  * @param publicKey
+ * @param secondSignature
+ * @param secondPublicKey
```
Comment (willclarktech) [inline src/crypto/sign.js]: Actually these Yuidoc annotations are incorrect - there's just one parameter.

File: test/crypto/sign.js [hunk 2]
```diff
@@ -115,8 +115,22 @@ ${defaultSignature}
 		});
 	});
 
-	describe('#signMessageWithTwoSecrets', () => {
-		it('should signTransaction a message using two secrets', () => {
+	describe('signTransaction and print messages with two secrets', () => {
+		const signedMessageWithTwoSecretsExample = `
+-----BEGIN LISK SIGNED MESSAGE-----
+-----MESSAGE-----
+not secret message
```
Comment (willclarktech) [inline test/crypto/sign.js]: Prefer this from the variable `notSecretMessage`

File: test/crypto/sign.js [hunk 3]
```diff
@@ -115,8 +115,22 @@ ${defaultSignature}
...
+${defaultPublicKey}
+-----SECOND PUBLIC KEY-----
+${defaultSecondPublicKey}
+-----SIGNATURE-----
+${defaultSignature}
+-----SECOND SIGNATURE-----
+${defaultSecondSignature}
+-----END LISK SIGNED MESSAGE-----
+`.trim();
+		it('#signMessageWithTwoSecret should signTransaction using two secrets', () => {
```
Comment (willclarktech) [inline test/crypto/sign.js]: `signMessageWithTwoSecrets`

**Pull request merged**